plugin/NERD_commenter.vim

```diff
@@ -144,7 +144,7 @@ let s:delimiterMap = {
     \ 'desktop': { 'left': '#' },
     \ 'dhcpd': { 'left': '#' },
     \ 'diff': { 'left': '#' },
-    \ 'django': { 'left': '<!--', 'right': '-->', 'leftAlt': '{#', 'rightAlt': '#}' },
+    \ 'django': { 'left': '{% comment %}', 'right': '{% endcomment %}', 'leftAlt': '{% comment %}', 'rightAlt': '{% endcomment %}' },
```

Comment:
If there is not an alternative comment syntax available, please just don't set the alternate delimiters.

```suggestion
    \ 'django': { 'left': '{% comment %}', 'right': '{% endcomment %}' },
```

Reply:
An alternate comment used amongst django community members is `{# code #}` which I've updated